apps/commercetools/src/api/categories.ts

```diff
@@ -3,16 +3,20 @@ import { Pagination, Product } from '@contentful/ecommerce-app-base';
 import { ConfigurationParameters } from '../types';
 import { createClient } from './client';
 
-function categoryTransformer({ projectKey, locale }: ConfigurationParameters) {
+function categoryTransformer({ projectKey, locale, mcUrl }: ConfigurationParameters) {
   return (item: Category): Product => {
+    const merchantCenterBaseUrl =
+      mcUrl && mcUrl.length > 0 ? mcUrl : 'https://mc.europe-west1.gcp.commercetools.com';
```

Comment:
is there a chance for `mcUrl` to come through undefined? might be a little more readable with:

```typescript
const merchantCenterBaseUrl =
      mcUrl ?? 'https://mc.europe-west1.gcp.commercetools.com'; 
```

Reply:
yeah, sure. I just took exactly what was there in `product.ts` since I still don't know this app well. I think it can actually be both undefined and an empty string actually.

Reply:
Yeah just confirmed that, so we'll need to keep it like this. Good thinking though